### ISSUE TITLE
Preview execution of pure functions

### ISSUE THREAD

**Issue by jariji** [KristofferC/OhMyREPL.jl#328]
NodeJS repl shows previews of pure function results before pressing enter. Seems like a nice feature.

![image](https://github.com/KristofferC/OhMyREPL.jl/assets/96840304/6c5c1a3a-bdbd-4f0a-b070-aa40795f049b)

https://nodejs.org/en/blog/release/v12.17.0#repl-previews

**Comment by Moelf** [comment]
we should tag this as feature / enhancement

**Comment by KristofferC** [comment]
After #395, OhMyREPL is a thin layer over the native REPL (colorschemes + prompts), so this is out of scope here.
